beacon-chain/rpc/prysm/v1alpha1/validator/server.go

```diff
@@ -100,14 +100,13 @@ func (vs *Server) WaitForActivation(req *ethpb.ValidatorActivationRequest, strea
 	}
 
 	waitTime := time.Duration(params.BeaconConfig().SecondsPerSlot) * time.Second
-	timer := time.NewTimer(waitTime)
-	defer timer.Stop()
+	ticker := time.NewTicker(waitTime)
```

Comment:
I don't think we need to use a ticker here , instead using the New Timer because we only have a 1 time action. Wait for Activation only triggers if there are 0 active keys while accounts are updated. the action no longer needs to run afterwards

Reply:
ok that's fair, I think you're right on this and I was mistaken

Reply:
Thanks for this I'll go ahead and approve